lib/triangle_farms/calendar/event_api.ex

```diff
@@ -1,9 +1,14 @@
 defmodule TriangleFarms.Calendar.EventAPI do
   alias TriangleFarms.Calendar.EventAPI.Client
+  alias TriangleFarms.Calendar.Time
 
   def new do
-    {:ok, client} = Client.authorize
-    client
+    case Client.authorize do
```

Comment:
Added a case to highlight that this function didn't deal with all the return values of `Client.authorize`. Not sure if you were intended to just "let it crash" if you couldn't authorize or not.

Reply:
Yeah in this case I'd just want it to crash and burn :)